velox/exec/tests/utils/OperatorTestBase.cpp

```diff
@@ -32,6 +32,8 @@
 
 using namespace facebook::velox::common::testutil;
 
+DECLARE_bool(velox_enable_memory_usage_track_in_default_memory_pool);
```

Comment:
Unrelated change? In general, let's avoid using gflags.

Reply:
I want to track the spilling memory usage and verify in test. Next will report to Prestissimo for monitoring such as the peak spilling memory usage which might give us some insights on spilling workload and the corresponding memory capacity provision.  I will change this flag to a system config in followup. Thanks!

Reply:
Still seeing this gflag. It would be best to avoid introducing new gflags.